Ports/nettle/package.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+
+port='nettle'
+version='3.8'
+useconfigure=true
+files="https://ftp.gnu.org/gnu/nettle/nettle-${version}.tar.gz nettle-${version}.tar.gz 
+https://ftp.gnu.org/gnu/nettle/nettle-${version}.tar.gz.sig nettle-${version}.tar.gz.sig"
+auth_type=sig
+auth_import_key='343C2FF0FBEE5EC2EDBEF399F3599FF828C67298'
+auth_opts=("nettle-${version}.tar.gz.sig")
```

Comment:
Please validate files using `sha256` instead of `sig`.

Reply:
fixed